tests/ut/plugin_demo/customIdx/customIdx_config.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2019-2023 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License.
+
+#ifndef FLAT_CONFIG_H
+#define FLAT_CONFIG_H
+
+#include "knowhere/config.h"
+
+namespace demo {
+class CustomIdxConfig : public knowhere::BaseConfig {
```

Comment:
comments here are needed as well
What if I need to return strings, ints, doubles, arrays?
A demo needs to demonstrate it all